packages/flutter_driver/lib/src/health.dart

```diff
@@ -0,0 +1,53 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'message.dart';
+
+/// Requests an application health check.
+class GetHealth implements Command {
+  final String kind = 'get_health';
+
+  GetHealth();
+
+  static fromJson(_) => new GetHealth();
+
+  Map toJson() => const {};
+}
+
+/// Application health status.
+class Health extends Result {
+  Health(this.status) {
+    assert(status != null);
+  }
+
+  static Health fromJson(Map json) {
+    return new Health(HealthStatus.fromId(json['status']));
+  }
+
+  /// Health status
+  final HealthStatus status;
+
+  Map toJson() => {
+    'status': status.id
+  };
+}
+
+/// Application health status.
+///
+// Note, it's not an enum because we cannot rely on enum's.
```

Comment:
You know, I cannot recall anymore why I thought we couldn't use `enum`, so I refactored it to `enum` :)